plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Internal.hs

```diff
@@ -416,7 +417,7 @@ throwingDischarged
     -> t
     -> CekValue uni fun
     -> CekM uni fun s x
-throwingDischarged l t = throwingWithCause l t . Just . dischargeCekValue
+throwingDischarged l t = throwingWithCause l t . Just . etermToTerm . dischargeCekValue
```

Comment:
What do we gain by going back to Term here? Wouldn't it simpler that throwingwithcause operated directly on ETerms?

Reply:
Yeah, this is a leftover from when I switched to making the interface to the CEK machine just use `ETerm`s.